CollisionDetectionSystem/Interfaces/ITransponderReceiver.cs

```diff
@@ -1,13 +1,14 @@
 ﻿using System;
+using System.Collections.Generic;
 
 namespace CollisionDetectionSystem
 {
-	public delegate void DataDel(TransponderData data);
+	public delegate void DataDel( List<TransponderData> data);
 
 	public interface ITransponderReceiver
 	{
-		void ReceiveData(TransponderData data);
-		void PrepareDataForPost(TransponderData data);
+		void ReceiveData(List<TransponderData> data);
+		void PrepareDataForPost( List<TransponderData> data);
 		event DataDel PostDataEvent;
```

Comment:
A transponder is not what is tracking the planes.  https://en.wikipedia.org/wiki/Transponder_%28aeronautics%29
The transponder sits on an aircraft and sends out a ping when asked for a one.  We are mocking this situation (for now mocked for just our aircraft and one intruder to keep it simplistic).  

 Our transponderReceiver gets one ping from all the aircrafts around us because we are listening to their responses when asked by a radar sitting elsewhere.   What we are calling the DataProcessor was to keep a list of these data points by aircraft.

So, with that, I do see your point in that you want all the similar timestamps to trigger at the same time.  But, then we should NOT create a list but do a rapid fire of sending BroadCastDataEvent for the timestamps that are the same from both files to mimic real life.

Our data is pretty contrived, so to mimic that behavior might be as simple as to call BroadCastDataEvent for each file at the same time, and then doing the slight pause. 
So...if you are creating the lists by a minute's worth of timestamps, then in mockTransponder, call BroadCastDataEvent for each element in that list, but don't send a list to the transponderReceiver...would be my suggestion.